fearless/Common/Extension/Foundation/TimeInterval+Time.swift

```diff
@@ -8,4 +8,5 @@ extension TimeInterval {
     var seconds: TimeInterval { self / 1000 }
     var daysFromSeconds: Int { Int(self / Self.secondsInDay) }
     var hoursFromSeconds: Int { Int(self / Self.secondsInHour) }
+    var intervalsInDay: Int { self > 0.0 ? Int(Self.secondsInDay / self) : 0 }
```

Comment:
Here is a possibility for a bug that could be hard to find. I would suggest to throw an error if time interval is negative